cloudinit/sources/DataSourceOracle.py

```diff
@@ -171,10 +105,11 @@ class DataSourceOracle(sources.DataSource):
         sources.NetworkConfigSource.system_cfg,
     )
 
-    _network_config = sources.UNSET
+    _network_config = sources.UNSET  # type: dict
```

Comment:
Wasn't sure how to declare these types. If I don't declare anything, my type checking complains anywhere I use `_network_config` as a dict because `sources.UNSET` is a string. I still get a complaint on this line because I'm declaring a string as a dict, but it has quieted the rest of the warnings. If we were being strict it would be a Union[dict|str], but since we don't ever actually want to use it as a string, that also seemed wrong to me (and I didn't want to add the union import just for that).

*Shrug*. Something to think about if we start to incorporate more typing.